actors/common/src/main/java/org/sunbird/learner/util/UserUtility.java

```diff
@@ -118,7 +118,7 @@ private UserUtility() {}
     // Decrypt user basic info
     for (String key : userKeyToDecrypt) {
       if (userMap.containsKey(key)) {
-        if (JsonKey.EMAIL.equalsIgnoreCase(key) || JsonKey.PHONE.equalsIgnoreCase(key) || JsonKey.PREV_USED_EMAIL.equalsIgnoreCase(key)|| JsonKey.PREV_USED_PHONE.equalsIgnoreCase(key)) {
+        if (JsonKey.EMAIL.equalsIgnoreCase(key) || JsonKey.PHONE.equalsIgnoreCase(key) || JsonKey.PREV_USED_EMAIL.equalsIgnoreCase(key)|| JsonKey.PREV_USED_PHONE.equalsIgnoreCase(key) || JsonKey.RECOVERY_PHONE.equalsIgnoreCase(key) || JsonKey.RECOVERY_EMAIL.equalsIgnoreCase(key)) {
```

Comment:
Use static  List of constant String and then find key in that 
list.contians methos